swift/files/liberty/dispersion.conf

```diff
@@ -0,0 +1,44 @@
+[dispersion] 
+{%- if pillar.swift.common.dispersion is defined %}
+{%- for k, v in pillar.swift.common.dispersion.iteritems() %}
+{{ k }} = {{ v }}
```

Comment:
Code that allow to do define anything, this is not correct metadata-driven approach.